roachtest: Update pgx block list

spilchen opened the pull request:
The pgx test v5.TestBeginReadOnly previously expected an error when executing:

```
BEGIN READ ONLY
CREATE TABLE foo(id serial primary key)
```

Due to the recent autocommit_before_ddl change in #141145, the transaction is auto-committed before the CREATE TABLE, allowing the command to succeed. This update adjusts the block list so that we ignore this failed test.

I will backport this in: #141987 #141851.

Epic: none
Release note: none

Closes #141903

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/142043)
<!-- Reviewable:end -->

spilchen committed:
roachtest: Update pgx block list

The pgx test v5.TestBeginReadOnly previously expected an error when
executing:

```
BEGIN READ ONLY
CREATE TABLE foo(id serial primary key)
```

Due to the recent autocommit_before_ddl change in #141145, the
transaction is auto-committed before the CREATE TABLE, allowing the
command to succeed. This update adjusts the block list so that we
ignore this failed test.

Epic: none
Release note: none

Closes #141903

rafiss reviewed (approved):
lgtm!

spilchen commented:
TFTR!

bors r+
